Projects/Server/Serialization/GenericPersistence.cs

```diff
@@ -73,8 +73,8 @@ void Deserialize(string savePath)
                 catch (Exception e)
                 {
                     Utility.PushColor(ConsoleColor.Red);
-                    Persistence.WriteConsoleLine($"***** Bad deserialize of {name} *****");
-                    Persistence.WriteConsoleLine(e.ToString());
+                    Console.WriteLine($"***** Bad deserialize of {name} *****");
```

Comment:
Yes, I changed to log always to console because it is part of the interactive world load (i.e. right after this user is asked whether to continue loading or not). IMHO doesn't make much sense to prompt the user via console but asking her elsewhere ^^u